utils/graphite-rollup/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ target_include_directories(
     ${ClickHouse_SOURCE_DIR}/contrib/double-conversion ${ClickHouse_SOURCE_DIR}/contrib/dragonbox/include
     ${ClickHouse_SOURCE_DIR}/contrib/fmtlib/include
     ${ClickHouse_SOURCE_DIR}/contrib/cityhash102/include
-    ${RE2_INCLUDE_DIR} ${CMAKE_BINARY_DIR}/contrib/re2_st
+    ${RE2_INCLUDE_DIR} ${CMAKE_BINARY_DIR}/contrib/re2-cmake/re2_st
```

Comment:
@zhanghuajieHIT why this is necessary ?

Reply:
There is `contrib/re2_st` in 21.8.
But, `re2_st` is replaced by `re2-cmake`  in master. I think it should be modified to `${CMAKE_BINARY_DIR}/contrib/re2-cmake`
Otherwise, it will make compile error:
`cc1plus: error: /home/code/github/clickhouse_github/build/contrib/re2_st: No such file or directory [-Werror=missing-include-dirs]`